test/integration/040_init_test/test_init.py

```diff
@@ -0,0 +1,37 @@
+
+from test.integration.base import DBTIntegrationTest, use_profile
+import os
+import shutil
+
+
+class TestInit(DBTIntegrationTest):
+    def tearDown(self):
+        project_name = self.get_project_name()
+
+        if os.path.exists(project_name):
+            shutil.rmtree(project_name)
+
+        DBTIntegrationTest.tearDown(self)
```

Comment:
`super(TestInit, self).tearDown()` is the canonical syntax for this, though it isn't very important here.